invenio_app_rdm/communities_ui/views/ui.py

```diff
@@ -94,6 +94,12 @@ def create_ui_blueprint(app):
         view_func=communities_home,
     )
 
+    blueprint.add_url_rule(
+        routes["community-static-page"],
+        view_func=community_static_page,
+        defaults={"page_url": routes["community-static-page"]},
+    )
+
```

Comment:
Question:
Is there a specific reason for setting the `defaults` for `page_url` to the route itself in `add_url_rule`? 
Could this be redundant?

Reply:
You're right it is redundant and leftover.